src/main/java/uk/gov/hmcts/reform/cwrdapi/util/MappingField.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface MappingField {
+
+    String columnName() default "";
+
+    Class clazz() default Object.class;
+
+    int noOfCollectionProperties() default 0;
+
+    String[] childes()  default "";
```

Comment:
Looks like this is not used anywhere.

Reply:
Yes same need to be in sync with 1952.

Reply:
Thanks. can the variable be updated to `children` if the variable meant to be no. of child class ?